osu.Game/OsuGame.cs

```diff
@@ -407,17 +400,17 @@ protected override void LoadComplete()
                 idleTracker = new GameIdleTracker(6000)
             });
 
-            dependencies.Cache(backgroundStack);
-
             screenStack.ScreenPushed += screenPushed;
             screenStack.ScreenExited += screenExited;
 
-            loadComponentSingleFile(osuLogo, logoContainer.Add);
-
-            loadComponentSingleFile(new Loader
+            loadComponentSingleFile(osuLogo, logo =>
             {
-                RelativeSizeAxes = Axes.Both
-            }, screenStack.Push);
+                logoContainer.Add(logo);
+                screenStack.Push(new Loader
```

Comment:
needs a comment as to why this is done here (also newline above).

Reply:
this comment is wrong. loader is not being created synchronously, nor is it done to successfully DI the background stack.

the comment is supposed to explain why it's laded after the logo, which is the true intent here, no?